ui/raidboss/data/06-ew/raid/p9n.txt

```diff
@@ -1,6 +1,259 @@
+194.5 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+197.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+198.8 "--sync--" sync / 1[56]:[^:]*:Kokytos:8144:/
+
+
+
+# Behemoth phase 1
+203.5 "Ravening (Behemoth)" sync / 1[56]:[^:]*:Kokytos:811A:/
+220.8 "Charybdis" sync / 1[56]:[^:]*:Kokytos:8132:/
+221.8 "--sync--" sync / 1[56]:[^:]*:Kokytos:8133:/
+226.9 "Beastly Roar" sync / 1[56]:[^:]*:Kokytos:8138:/
+229.0 "Beastly Bile" sync / 1[56]:[^:]*:Kokytos:813D:/
+231.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:813E:/
+238.1 "Pulverizing Pounce" sync / 1[56]:[^:]*:Kokytos:813F:/
+240.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8140:/
+243.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:8140:/
+253.4 "Gluttonous Rampage" sync / 1[56]:[^:]*:Kokytos:8139:/
+253.6 "Ravenous Bite 1" #sync / 1[56]:[^:]*:Kokytos:813A:/
+255.7 "Ravenous Bite 2" #sync / 1[56]:[^:]*:Kokytos:813A:/
+257.9 "Touchdown" sync / 1[56]:[^:]*:Kokytos:8137:/
+258.1 "--sync--" sync / 1[56]:[^:]*:Kokytos:8146:/
+268.8 "Comet" sync / 1[56]:[^:]*:Kokytos:8134:/
+269.8 "--sync--" sync / 1[56]:[^:]*:Comet:8135:/
+277.8 "--sync--" sync / 1[56]:[^:]*:Kokytos:813B:/
+278.8 "--sync--" sync / 1[56]:[^:]*:Kokytos:813C:/
+279.8 "Ecliptic Meteor" sync / 1[56]:[^:]*:Kokytos:8188:/
+279.9 "--sync--" sync / 1[56]:[^:]*:Comet:8143:/
+285.9 "Burst" sync / 1[56]:[^:]*:Comet:8136:/
+286.0 "Pulverizing Pounce" sync / 1[56]:[^:]*:Kokytos:813F:/
+288.8 "Disgorge" sync / 1[56]:[^:]*:Kokytos:8126:/
+296.1 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+299.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+
+
+
+# Mage phase 2
+304.3 "Ravening (Mage)" sync / 1[56]:[^:]*:Kokytos:8118:/
+313.4 "--sync--" sync / 1[56]:[^:]*:Kokytos:811C:/
+316.5 "--sync--" sync / 1[56]:[^:]*:Kokytos:811C:/
+318.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:8144:/
+325.2 "Iceflame Summoning" sync / 1[56]:[^:]*:Kokytos:86E1:/
+330.1 "--sync--" sync / 1[56]:[^:]*:Kokytos:812[23]:/
+336.1 "--sync--" sync / 1[56]:[^:]*:Ice Sphere:872[9A]:/
+336.1 "--sync--" sync / 1[56]:[^:]*:Fire Sphere:872[78]:/
+336.1 "Sphere Shatter" sync / 1[56]:[^:]*:Ice Sphere:86E[45]:/
+336.1 "Explosion" sync / 1[56]:[^:]*:Fire Sphere:86E[23]:/
+340.7 "Iceflame Summoning" sync / 1[56]:[^:]*:Kokytos:86E1:/
+345.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:812[23]:/
+347.9 "--sync--" sync / 1[56]:[^:]*:Kokytos:811C:/
+351.6 "--sync--" sync / 1[56]:[^:]*:Ice Sphere:872[9A]:/
+351.6 "--sync--" sync / 1[56]:[^:]*:Fire Sphere:872[78]:/
+351.6 "Sphere Shatter" sync / 1[56]:[^:]*:Ice Sphere:86E[45]:/
+351.6 "Explosion" sync / 1[56]:[^:]*:Fire Sphere:86E[23]:/
+356.0 "Dualspell" sync / 1[56]:[^:]*:Kokytos:811D:/
+359.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:812[23]:/
+361.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:811C:/
+363.8 "Blizzard III" sync / 1[56]:[^:]*:Kokytos:(811F|8121):/
+363.8 "Fire III" sync / 1[56]:[^:]*:Kokytos:(811E|8120):/
+364.4 "--sync--" sync / 1[56]:[^:]*:Kokytos:811C:/
+370.5 "Global Spell" sync / 1[56]:[^:]*:Kokytos:8141:/
+371.4 "Global Spell" sync / 1[56]:[^:]*:Kokytos:8142:/
```

Comment:
Can you `-ii 8142` as well? `8141` is the cast, and `8142` is the damage on the party.  There's only one `Global Spell` occurring here.

Reply:
`-ii` or mark them as `--sync--`? There's a few other cases of this, e.g. `Beastly Bile` which casts on `813D` and damage is on `813E`, and `Archaic Demolish` is `812F`/`8130`. Most of them I marked the damage one as `--sync--`, including some other `Global Spell` casts, I had just missed this one.

Reply:
Typically (and this isn't to say any of us have been particularly consistent about it) if it's a dual situation like this we will ignore the cast entirely. "More syncs is better" has some truth to it, but on the flip side, we want the timeline to be as human-readable as possible. Fewer interspersed `"--sync--"` entries would help with that.

(Note that this does not necessarily hold in situations where there's a cast and then *multiple* follow-ups. In that situation we would usually take it case-by-case.)

Reply:
Yeah, that's my feeling too, is that there's some balance in having the timeline be readable with too many `--sync--`s in it.  Really the balance is "are there enough abilities to keep the timeline on track" vs "is the timeline readable", which for me is just something every couple of seconds or so.  If there's a bigger break with some syncs in the middle, then definitely keep those.  If it's cast and damage, I don't think you need both.

In other words, I would ignore in this case, because the cast a second earlier more than covers it.

Reply:
Ignored this with the other `-ii`s as mentioned in the other comment.